sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1236,9 +1236,9 @@ static void *_gnrc_netif_thread(void *args)
     netif->pid = sched_active_pid;
     /* setup the link-layer's message queue */
     msg_init_queue(msg_queue, _NETIF_NETAPI_MSG_QUEUE_SIZE);
+    dev->context = netif;
     /* register the event callback with the device driver */
     dev->event_callback = _event_cb;
-    dev->context = netif;
```

Comment:
Still unrelated move?

Reply:
Ping?

Reply:
Yeah, I have to check this, probably still unrelated, but I have a hunch that I switched the order intentionally

Reply:
Always document your work. Future you and your peers will thank you. ;-)